README.md

```diff
@@ -9,11 +9,14 @@ We run the nginx proxy inside a Fargate container for all of our apps.
 To update and deploy a new nginx-forward-proxy container:
```

Comment:
Was this copied and pasted from the `nginx-forward-proxy` README?

Reply:
Looks like it. I'll fix